client/src/components/App.jsx

```diff
@@ -34,10 +51,13 @@ const App = (props) => {
         <div className="top_bar col-8">Top Bar Goes Here</div>
       </div>
       <Overview productId={ currentProductId }/>
+      
```

Comment:
Are these spaces here just for readability?

Reply:
Mainly ESlinting issues. Also I believe my auto formatting command does this as well.